### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by MisterVants**
Improvements to documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,30 +1,38 @@
...
+3. Inside your project, you will see a Project ID string. Make sure to keep it well secured.
 
-Apache 2.0
+### Project ID
+Anyone who registers a project into the cloud application will be granted a Project ID associated with that project. The project ID allows you to start using the relay network immediately, and is needed during SDK initialization.
 
+## Documentation & Usage
+To dive deeper into the protocol concepts, check out our documentation:
+- [Getting started with wallet integration](https://docs.walletconnect.com/2.0/quick-start/wallets/swift)
+- [Protocol Documentation](https://docs.walletconnect.com/2.0/protocol/client-communication)
```
Comment (chadyj) [inline README.md:28]: 404 link

File: README.md [hunk 2]
```diff
@@ -1,30 +1,38 @@
...
+2. Sign in and click the "+ New Project" button. Give a name to your new project.
+3. Inside your project, you will see a Project ID string. Make sure to keep it well secured.
 
-Apache 2.0
+### Project ID
+Anyone who registers a project into the cloud application will be granted a Project ID associated with that project. The project ID allows you to start using the relay network immediately, and is needed during SDK initialization.
 
+## Documentation & Usage
+To dive deeper into the protocol concepts, check out our documentation:
+- [Getting started with wallet integration](https://docs.walletconnect.com/2.0/quick-start/wallets/swift)
```
Comment (chadyj) [inline README.md:27]: 404 link

File: README.md [hunk 3]
```diff
@@ -1,30 +1,38 @@
...
+- [Protocol Documentation](https://docs.walletconnect.com/2.0/protocol/client-communication)
+- [Beginner guide to WalletConnect v2.0 for iOS Developers](https://medium.com/walletconnect/beginner-guide-to-walletconnect-v2-0-for-swift-developers-4534b0975218)
+- [Glossary](https://docs.walletconnect.com/2.0/protocol/glossary)
```
Comment (chadyj) [inline README.md:30]: 404

File: README.md (hunk 1 above)
Comment (MisterVants) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (MisterVants) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (MisterVants) [reply README.md]: Fixed

**Comment by Mike117687** [comment]
merge-these-changes

**Review by flypaper0 — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -24,10 +24,10 @@ Anyone who registers a project into the cloud application will be granted a Proj
 
 ## Documentation & Usage
 To dive deeper into the protocol concepts, check out our documentation:
-- [Getting started with wallet integration](https://docs.walletconnect.com/2.0/quick-start/wallets/swift)
-- [Protocol Documentation](https://docs.walletconnect.com/2.0/protocol/client-communication)
+- [Getting started with wallet integration](https://docs.walletconnect.com/2.0/swift/sign/installation)
+- [Protocol Documentation](https://github.com/WalletConnect/walletconnect-specs)
 - [Beginner guide to WalletConnect v2.0 for iOS Developers](https://medium.com/walletconnect/beginner-guide-to-walletconnect-v2-0-for-swift-developers-4534b0975218)
-- [Glossary](https://docs.walletconnect.com/2.0/protocol/glossary)
+- [Glossary](https://docs.walletconnect.com/2.0/introduction/glossary)
```
[the remark this answers is not in the document]
Comment (MisterVants) [reply README.md:30]: Indeed, let's remove for now

**Pull request merged**